h2o-core/src/test/java/water/junit/FatalCrashListener.java

```diff
@@ -0,0 +1,19 @@
+package water.junit;
```

Comment:
I would remove this from this PR - there is another PR with ongoing discussion. Let's not hold this pr back because of this change.

Reply:
sure, this is here just for me to see quickly the reason, not meant to stay here, sorry for confusion.